sys/linux/sys.txt

```diff
@@ -293,9 +293,12 @@ utimes(filename ptr[in, filename], times ptr[in, itimerval])
 futimesat(dir fd_dir, pathname ptr[in, filename], times ptr[in, itimerval])
 utimensat(dir fd_dir, pathname ptr[in, filename], times ptr[in, itimerval], flags flags[utimensat_flags])
 
-fork() pid (breaks_returns)
-clone(flags flags[clone_flags], sp buffer[in], parentid ptr[out, int32], childtid ptr[out, int32], tls buffer[in]) (breaks_returns)
-clone3(args ptr[in, clone_args], size bytesize[args]) pid (breaks_returns)
+syz_fork() pid
+syz_clone3(args ptr[in, clone_args], size bytesize[args]) pid
```

Comment:
is it worth to introduce syz_clone too?

Reply:
For Linux kernel, doesn't seem worth it - except for a tiny wrapper function we reach absolutely the same code.

Above you mentioned "This will not work for gVisor that doesn't support clone3, ....". Did you mean a hypothetical situation/older gVisor or does gVisor not support clone3 now?

It's not that it will be particularly hard to add syz_clone, but it just seems to be an unnecessary complication, given that older kernels are only becoming less interesting for us.

Reply:
gVisor doesn't support clone3 now and we don't have a plan to support it in a near future. If it isn't a lot of work, we would like to have it. If you don't want to spend time on this, I can add it when you merge this pr.

Reply:
Hmm, if gVisor doesn't support that, then it makes it more reasonable to indeed add the syz_clone call (and maybe delete syz_fork in that case?). I'll add it to the PR tomorrow and hopefully we'll merge it all soon.